### ISSUE TITLE
Support Blender 2.73-2.75

### ISSUE THREAD

**Issue by Niondir** [indyjo/bitwrk#102]
Is it sufficient to just add the detection for Blender 2.7.3 to the python file?

like:
```
def get_blender_version():
    proc = subprocess.Popen([BLENDER_BIN, '-v'], stdout=subprocess.PIPE)
    output, _ = proc.communicate()
    if b"Blender 2.69" in output:
        return "2.69"
    elif b"Blender 2.70 (sub 0)" in output:
        return "2.70"
    elif b"Blender 2.71 (sub 0)" in output:
        return "2.71"
    elif b"Blender 2.72 (sub 0)" in output:
        return "2.72"
	elif b"Blender 2.73 (sub 0)" in output:
        return "2.73"
    else:
        raise RuntimeError("Blender version could not be detected.\n"
                           + "This version of blender-slave.py will detect Blender versions "
                           + "2.69 up to 2.73.")
```

**Comment by indyjo** [comment]
Unfortunately no, at least the server has to accept the new article. There is a positive list in server_bid.go, right above an enlightening comment:
```go
	case "fnord", "snafu", "foobar",
		"net.bitwrk/gorays/0",
		"net.bitwrk/blender/0/2.69/512M",
		"net.bitwrk/blender/0/2.69/2G",
		"net.bitwrk/blender/0/2.69/8G",
		"net.bitwrk/blender/0/2.69/32G",
		"net.bitwrk/blender/0/2.70/512M",
		"net.bitwrk/blender/0/2.70/2G",
		"net.bitwrk/blender/0/2.70/8G",
		"net.bitwrk/blender/0/2.70/32G",
		"net.bitwrk/blender/0/2.71/512M",
		"net.bitwrk/blender/0/2.71/2G",
		"net.bitwrk/blender/0/2.71/8G",
		"net.bitwrk/blender/0/2.71/32G",
		"net.bitwrk/blender/0/2.72/512M",
		"net.bitwrk/blender/0/2.72/2G",
		"net.bitwrk/blender/0/2.72/8G",
		"net.bitwrk/blender/0/2.72/32G":
		// TODO: add real article management
```

I usually take the opportunity to include some more blender features. And then there have to be a couple of workers running the new version. How urgent is it? ;-)

**Comment by Niondir** [comment]
It's easier to get started with it. I just downloaded blender and got the latest version. It actually worked when I edited the py script to pretend to have a older version ;) So maybe it's a good idea to assume that new Blender versions are backwards compatible - but I do not know if that assumption holds. Maybe for the minor versions.

**Comment by indyjo** [comment]
I agree that workers could theoretically accept jobs from previous blender versions, but client-side is a little more involved: In order to limit a job's computational complexity, both add-on (render_bitwrk.py) and worker script (blender-slave.py) have to have some understanding of Blender features. That's why I like to take some time to support a new version: I have to play around with the new features and learn how to calculate their effect on rendering complexity.

TLDR; will get to it soon.

**Comment by indyjo** [comment]
Sorry @Niondir it took me so long, due to real life issues I couldn't put as much time into BitWrk as I would have liked. Moved houses, changed jobs, lots of stress.
